source/fundamentals/connection.txt

```diff
@@ -101,12 +101,20 @@ redundancy and high data availability.
 To connect to a replica set deployment, specify the hostname (or IP address)
 and port numbers of one or more members of the replica set separated by commas.
 By default, specifying the hostname and port number of a single MongoDB
-instance will cause the driver to only connect to the MongoDB instance provided.
-However, if you specify the name of the replica set using the
-``replicaSet`` parameter, the driver creates a replica set connection
-and will discover all of the members of the set. Specifying multiple hosts
-also causes the driver to create a replica set connection and discover
-all members of the set.
+instance only connects to the specified member of the replica set.
+However, you can automatically discover and connect all members of the
+replica set by creating a replica set connection in several different
```

Comment:
The `in several different ways` maybe needs to be set off by a comma?  It reads to me like it applies to `by creating a replica set connection` rather than the main clause

Reply:
Removed the confusing clause.